lib/matplotlib/style/core.py

```diff
@@ -71,18 +96,18 @@ def use(style):
 
     for style in styles:
         if not cbook.is_string_like(style):
-            mpl.rcParams.update(style)
+            _apply_style(style)
             continue
         elif style == 'default':
             mpl.rcdefaults()
```

Comment:
There is a bit of a disconnect here. rcdefaults() is more than just style stuff, but it doesn't make sense to blacklist entries there, either, does it?